lab-yunjoo/.gitignore

```diff
@@ -0,0 +1,95 @@
+# Created by https://www.gitignore.io/api/osx,linux,node,vim
```

Comment:
This is a good base template, but you need to add your own stuff to it. If you look at this pull request, it says that 146 files were changed because you didn't git ignore the data/note directory.